eclipse-scout-core/src/focus/FocusContext.js

```diff
@@ -286,6 +290,10 @@ export default class FocusContext {
     elementToFocus.focus({
       preventScroll: scout.nvl(options.preventScroll, false)
     });
+    // If possible and requested, select the text content
+    if (elementToFocus instanceof HTMLInputElement && options.selectText) {
```

Comment:
Unfortunately, instanceof does not work for these HTML elements if used in a popup window and probably also in iframes.

Reply:
The code now checks if a functions called "select" exists on the element instead.